Lib/test/libregrtest/main.py

```diff
@@ -106,7 +106,11 @@ def __init__(self, ns: Namespace, _add_python_opts: bool = False):
         self.fail_rerun: bool = ns.fail_rerun
         self.forever: bool = ns.forever
         self.randomize: bool = ns.randomize
-        self.random_seed: int | None = ns.random_seed
+        self.random_seed: int =  (
+            ns.random_seed
+            if ns.random_seed is not None
+            else random.randrange(100_000_000)
```

Comment:
While we are here... Let's be crazy, and use 29.9 bits of entropy for the seed!!! 9 digits.

```suggestion
            else random.randrange(1_000_000_000)
```

You should update test_regrtest for that:

```
        self.assertTrue(0 <= randseed <= 100_000_000, randseed)
```

Honestly, let's just remove the maximum value, no?

Reply:
`pytest-randomly` does this: `random.Random().getrandbits(32)` https://github.com/pytest-dev/pytest-randomly/blob/412b64004dd8820022267e3f687a2e651d7e70c3/src/pytest_randomly/__init__.py#L292

For me, it is cleaner.

Reply:
Woooooow, 32 bits of entropy!? Amazing :-D Sure, go for getrandbits(32).